net/coova-chilli/Makefile

```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=coova-chilli
 PKG_VERSION:=1.6
-PKG_RELEASE:=$(AUTORELEASE)
+PKG_RELEASE:=$(AUTORELEASE).1
```

Comment:
uhhh

Reply:
Is there any better solution how to bump package version without introducing dummy commit?

Reply:
No. I wonder if AUTORELEASE should get treewide functionality.

Reply:
IMO this is not relevant in context of this PR and can't be backported to 22.03 and 21.02 either.